inference-engine/tests/functional/plugin/myriad/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -32,6 +32,8 @@ std::vector<std::string> disabledTestPatterns() {
         // TODO: Issue 26090
         ".*DSR_GatherStaticDataDynamicIdx.*f32.*1.3.200.304.*",
         // TODO: Issue 46755
-        ".*DSR_GatherElements.*"
+        ".*DSR_GatherElements.*",
+        // TODO: Issue 48183
+        R"(.*CTCGreedyDecoderSeqLen.*?\(1.1.1\).*)",
```

Comment:
Does it relate to the 1-element input issue?

Reply:
As I understood, yes.